test/integration/model/findByPks.test.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const chai = require('chai'),
+  Sequelize = require('../../../index'),
+  expect = chai.expect,
+  Support = require('../support'),
+  DataTypes = require('../../../lib/data-types'),
+  current = Support.sequelize;
+
+describe(Support.getTestDialectTeaser('Model'), () => {
+  beforeEach(function() {
+    this.User = this.sequelize.define('User', {
+      username: DataTypes.STRING,
+      secretValue: DataTypes.STRING,
+      data: DataTypes.STRING,
+      intVal: DataTypes.INTEGER,
+      theDate: DataTypes.DATE,
+      aBool: DataTypes.BOOLEAN,
+      binary: DataTypes.STRING(16, true)
+    });
+
+    return this.User.sync({ force: true });
+  });
+
+  describe('findByPks', () => {
+    if (current.dialect.supports.transactions) {
```

Comment:
The tests for `findByPks` should not involve transactions, please rewrite the tests removing transaction stuff